onnxruntime/core/optimizer/qdq_transformer/selectors_actions/qdq_actions.cc

```diff
@@ -94,8 +94,35 @@ struct SetOptionalZeroPoint {
   static void UpdateNodes(Graph&, const NodesToOptimize& selected_nodes);
 
  private:
-  static const ONNX_NAMESPACE::TensorProto optional_zero_point_int8_;
-  static const ONNX_NAMESPACE::TensorProto optional_zero_point_uint8_;
+  //We assume this function won't fail
+  static const ONNX_NAMESPACE::TensorProto init_optional_zero_point_int8() {
+    const char* const name = "b33fd0fa-cd7b-4b10-ae5a-df64cabfe1f8";  // guid as arbitrary name to provide a unique value
+    ONNX_NAMESPACE::TensorProto tensor_proto;
+    tensor_proto.set_name(name);
+    tensor_proto.set_data_type(ONNX_NAMESPACE::TensorProto_DataType_INT8);
+    tensor_proto.set_raw_data(std::vector<int8_t>{0}.data(), sizeof(int8_t));
+
+    return tensor_proto;
+  };
+
+  //We assume this function won't fail
+  static const ONNX_NAMESPACE::TensorProto init_optional_zero_point_uint8() {
+    const char* const name = "b33f88f7-c464-43e3-8692-97ac832bb14a";  // guid as arbitrary name to provide a unique value
+    ONNX_NAMESPACE::TensorProto tensor_proto;
+    tensor_proto.set_name(name);
+    tensor_proto.set_data_type(ONNX_NAMESPACE::TensorProto_DataType_UINT8);
+    tensor_proto.set_raw_data(std::vector<uint8_t>{0}.data(), sizeof(uint8_t));
```

Comment:
>std::vector<uint8_t>{0}.data(), sizeof(uint8_t) [](http://example.com/codeflow?start=30&length=47)

This would do new/delete. std::array on a stack is a better choice?